tests/test_routes.py

```diff
@@ -177,3 +178,106 @@ def test_get_wishlist_list(self):
         data = resp.get_json()
         self.assertEqual(len(data), 5)
         self.assertEqual(data, wishlist_array)
+
+    ######################################################################
+    #  W I S H L I S T   I T E M   T E S T   C A S E S   H E R E
+    ######################################################################
+    def test_create_wishlist_item(self):
+        """It should create a new Wishlist Item and associate it with a specific Wishlist"""
+        # Create a Wishlist to associate the item with
+        wishlist = WishlistFactory()
+        resp_wishlist = self.client.post(BASE_URL, json=wishlist.serialize())
+        self.assertEqual(resp_wishlist.status_code, status.HTTP_201_CREATED)
+        wishlist_data = resp_wishlist.get_json()
+
+        # Create a Wishlist Item
+        wishlist_item = WishlistItemFactory()
+        resp = self.client.post(
+            f'{BASE_URL}/{wishlist_data["id"]}/items',
+            json=wishlist_item.serialize(),
+            content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+
+        # Make sure location header is set (part of RESTful api definition)
+        location = resp.headers.get("Location", None)
+        self.assertIsNotNone(location)
+
+        # Check the data is correct (response from db)
+        new_wishlist_item = resp.get_json()
+        self.assertEqual(
+            new_wishlist_item["wishlist_id"],
+            wishlist_data["id"],
+            "Wishlist Item is not associated with the correct Wishlist",
+        )
+        self.assertEqual(
+            new_wishlist_item["product_id"],
+            wishlist_item.product_id,
+            "Product Id does not match",
+        )
+        self.assertEqual(
+            new_wishlist_item["product_name"],
+            wishlist_item.product_name,
+            "Product Name does not match",
+        )
+        self.assertEqual(
+            new_wishlist_item["product_price"],
+            str(wishlist_item.product_price),
+            "Product Price does not match",
+        )
+        self.assertEqual(
+            new_wishlist_item["quantity"],
+            wishlist_item.quantity,
+            "Quantity does not match",
+        )
+        self.assertEqual(
+            datetime.strptime(new_wishlist_item["created_date"], '%a, %d %b %Y %H:%M:%S GMT').date(),
```

Comment:
You might consider deserializing the new_wishlist_item and then comparing is created_date against the wishlist_item.created_date.  (Though this should work as is).